File: lib/models/object/mark.dart
class Mark {
  final String ID;
  final String MaSinhVien;
  final String MaMon;
  final String TenMon;
  final String SoTinChi;
  final String CC;
  final String KT;
  final String THI;
  final String TKHP;
  final String DiemChu;

  Mark(
      {this.ID,
      this.MaSinhVien,
      this.MaMon,
      this.TenMon,
      this.SoTinChi,
      this.CC,
      this.KT,
      this.THI,
      this.TKHP,
      this.DiemChu});

  factory Mark.fromJson(Map<String, dynamic> json) {
    return Mark(
        ID: json['ID'].toString(),
        MaSinhVien: json['MaSinhVien'],
        MaMon: json['MaMon'],
        TenMon: json['TenMon'],
        SoTinChi: json['SoTinChi'].toString(),
        CC: json['CC'].toString(),
        KT: json['KT'].toString(),
        THI: json['THI'].toString(),
        TKHP: json['TKHP'].toString(),
        DiemChu: json['DiemChu'].toString());
  }
}

File: ios/Runner/Model/Mark.swift
//
//  Mark.swift
//  Runner
//
//  Created by TBM on 6/17/19.
//  Copyright © 2019 The Chromium Authors. All rights reserved.
//
/*
 private int ID;
 private String MaSinhVien;
 private String MaMon;
 private String TenMon;
 private String SoTinChi;
 private String CC;
 private String KT;
 private String THI;
 private String TKHP;
 private String DiemChu;
 */

import Foundation
import RealmSwift
class Mark:  Object ,Codable{
    //những biến có dấu ? là những biến ko bắt buộc khi decode json phải có, có thể gán nó sau khi deocde
    @objc dynamic var ID:String? = ""
    @objc dynamic var MaSinhVien:String? = ""
    @objc dynamic var MaMon = ""
    @objc dynamic var TenMon:String? = ""
    @objc dynamic var SoTinChi:String? = ""
    @objc dynamic var CC = ""
    @objc dynamic var KT = 0.0
    @objc dynamic var THI = ""
    @objc dynamic var TKHP = ""
    @objc dynamic var DiemChu = ""
    override static func primaryKey() -> String? {
        return "ID"
    }
}
